CreditManagement/migrations/usercredit_view.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.1.5 on 2019-01-22 13:07
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+from General.DBView_migration import CreateView
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('UserDetails', '0001_initial'),
+        ('CreditManagement', '0001_initial'),
+    ]
+
+    operations = [
+        CreateView(
+            name='UserCredit',
+            fields=[
+                ('user', models.OneToOneField(db_column='id', on_delete=django.db.models.deletion.DO_NOTHING, primary_key=True, serialize=False, to=settings.AUTH_USER_MODEL)),
+                ('balance', models.DecimalField(blank=True, db_column='balance', decimal_places=2, max_digits=6, null=True)),
+            ],
```

Comment:
I saw that link, though felt some information to be missing (what exactly should the remainder of the migration file look like).
An earlier version did indeed use a normal table with a custom manager, it worked, but it could give the impression the contents of the table were used, which wasn't. That's why I researched the implementation of views directly.

Reply:
The previous style implementation is currently still implemented on 'PendingDiningTransaction', dus je kan daar kijken als je wilt. Het nadeel is (daar kwam ik net achter), als je bijvoorbeeld een user object verwijderd, dan knalt hij tegen een niet bestaande tabel en word de user daardoor niet verwijderd.

Reply:
Kun je de code die niet gebruikt wordt verwijderen?